connection.js

```diff
@@ -772,6 +772,7 @@ class Connection {
                 this.respond(500, msg || "Unrecognized command");
                 break;
             case constants.denydisconnect:
+            case constants.denysoftdisconnect:
```

Comment:
Shouldn't `denysoftdisconnect` disconnect after 421 not 521?

Reply:
Also no handler for plain `denysoft`.

Reply:
@smfreegard  said "The RFC isn't really very clear on unrecognized commands resulting in anything other than a 5xx". Thus, 5xx and not 4xx. Also, sending unrec commands is bad, especially when a plugin knows about it and explicitly requests client to disconnect.
Plain denysoft handled by `default` case.